app/Models/Reaction.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class Reaction extends Model
+{
+    use HasFactory;
+
+    protected $fillable = ['type', 'post_id', 'user_id'];
```

Comment:
Can be replaced with `guarded` (which is the opposite of `fillable`):
```suggestion
    protected $guarded = ['id'];
```

Reply:
oh that's great